src/page-modules/contact/travel-guarantee/index.tsx

```diff
@@ -0,0 +1,436 @@
+import FormSelector from './form-selector';
+import { SectionCard } from '../components/section-card';
+import { ComponentText, PageText, useTranslation } from '@atb/translations';
+import { useMachine } from '@xstate/react';
+import { fetchMachine } from './travelGuaranteeFormMachine';
+import { Input } from '../components/input';
+import { TransportModeType } from '@atb-as/config-specs';
+import { useLines } from '../lines/use-lines';
+import { FormEventHandler } from 'react';
+import style from '../contact.module.css';
+import { Button } from '@atb/components/button';
+import Select from '../components/input/select';
+import { Line } from '..';
+import { Checkbox } from '../components/input/checkbox';
+import ErrorMessage from '../components/input/error-message';
+
+export const RefundForm = () => {
+  const { t } = useTranslation();
+  const { getLinesByMode, getQuaysByLine } = useLines();
+  const [state, send] = useMachine(fetchMachine);
+
+  const onSubmit: FormEventHandler<HTMLFormElement> = async (e) => {
+    e.preventDefault();
+    send({ type: 'VALIDATE' });
+  };
+
+  return (
+    <form onSubmit={onSubmit}>
+      <FormSelector state={state} send={send} />
+      {(state.hasTag('taxi') || state.hasTag('car')) && (
+        <SectionCard
+          title={
+            PageText.Contact.travelGuarantee.refundTaxi.aboutYourTrip.title
+          }
+        >
+          <Select
+            label={t(
+              PageText.Contact.inputFields.transportMode.label,
+            ).toString()}
+            value={state.context.transportMode}
+            onChange={(value) =>
+              send({
+                type: 'UPDATE_FIELD',
+                field: 'transportMode',
+                value: value as TransportModeType,
+              })
+            }
+            error={
+              state.context?.errorMessages['transportMode']
+                ? t(state.context?.errorMessages['transportMode']?.[0])
+                : undefined
+            }
+            valueToText={(val: TransportModeType) =>
+              t(ComponentText.TransportMode.modes[val])
+            }
+            valueToId={(val: TransportModeType) => val}
+            options={['bus', 'water'] as TransportModeType[]}
+            placeholder={t(
+              PageText.Contact.inputFields.transportMode.optionLabel,
+            )}
+          />
+
+          {state.context.transportMode && (
```

Comment:
Thinking out loud here, but would it be better to have the input disabled if transport mode is not set, rather than not showing it?

Reply:
Maybe that is a good alternative. The reason for why it is hided is because transportMode must be set before being able to set Line. But i could take a quick look.

Reply:
Can look into it in another PR since we need to update the other forms as well